config/services.xml

```diff
@@ -4,26 +4,46 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <defaults autowire="true" autoconfigure="true"/>
+        <defaults autowire="true" autoconfigure="true" />
 
         <prototype namespace="Meilisearch\Bundle\Command\" resource="../src/Command" />
 
-        <service id="search.search_indexer_subscriber"
+        <service id="meilisearch.search_indexer_subscriber"
                  class="Meilisearch\Bundle\EventListener\DoctrineEventSubscriber"
                  public="true">
-            <argument type="service" id="search.service"/>
+            <argument type="service" id="meilisearch.service" />
+        </service>
+        <service id="search.search_indexer_subscriber" alias="meilisearch.search_indexer_subscriber">
+            <deprecated package="meilisearch/search-bundle" version="0.14">
+                The "%alias_id%" service alias is deprecated. Use "meilisearch.search_indexer_subscriber" instead.
```

Comment:
> Invalid characters found in deprecation template.
I think messages must go on same line as <deprecated> tag

Reply:
copy that, pushing rn